### PR TITLE
fix formatting

### PR DESCRIPTION
**Pull request by NukeManDan**
Redo of #528 as it broke things and #534 took it back out of prod.
See #528 for details of changes

### CODE REVIEW

**Comment by NukeManDan** [comment]
I pushed in #553 as a PATCH to get the CI to work (should properly fix to close #552)
I can confirm the site builds with 
```bash
yarn serve:fresh
# new term
yarn checklinks 
```
image

File: v3/docs/07-advanced/h-ss58/index.mdx [hunk 1]
```diff
@@ -89,14 +89,14 @@ For the shorter account-index formats, the extra byte represents a far greater p
 
 ## Simple/full address types and account/address/network identifiers
 
-[The registry](https://github.com/paritytech/ss58-registry) expresses the status of the account/address/network *identifiers*.
+[The registry](https://github.com/paritytech/ss58-registry) expresses the status of the account/address/network _identifiers_.
 
-*Identifiers* up to value 64 may be expressed in a *simple* format address, in which the least significant byte (LSB) of the identifier value is expressed as the first byte of the encoded address.
+_Identifiers_ up to value 64 may be expressed in a _simple_ format address, in which the least significant byte (LSB) of the identifier value is expressed as the first byte of the encoded address.
```
Comment (Eve-Parity) [inline v3/docs/07-advanced/h-ss58/index.mdx:94]: It's usually not a great idea to use italics for emphasis. Introduction of new terms should be **bolded**, code-based terms should be in `monospace`.

File: v3/how-to-guides/01-basics/a-pallet-integration/index.mdx [hunk 2]
```diff
@@ -5,6 +5,7 @@ keywords: basics, beginner, runtime
 version: '3.0'
 section: how to guides
 category: basics
+difficulty: 1
```
Comment (Eve-Parity) [inline v3/how-to-guides/01-basics/a-pallet-integration/index.mdx:8]: Took this out originally to remove clutter

File: v3/how-to-guides/01-basics/c-constants/index.mdx [hunk 3]
```diff
@@ -5,138 +5,141 @@ keywords: basics, runtime, configurable constants
 version: '3.0'
 section: how to guides
 category: basics
+difficulty: 1
 ---
 
-Declaring a constant value in a runtime is a useful tool to either define fixed values or define values that change dynamically according to some factor. This guide shows you how to create pallet constants that are used to reset a \`u32\` value in storage. This value, we'll call \`SingleValue\`, can also be modified using a method called \`add_value\`.
+Declaring a constant value in a runtime is a useful tool to either define fixed values or define values that change dynamically according to some factor.
```
Comment (Eve-Parity) [inline v3/how-to-guides/01-basics/c-constants/index.mdx:11]: Not sure why we're using `\` here...

**Review by Eve-Parity — COMMENTED** [review]
Looks good with the following considerations:
- Best practice is not to use italics for emphasis
- There are places where `/` is used and shouldn't be - i think this is legacy from when monospace font was needed within one of the CSS modules 
- I had removed the `difficulty` line from the front matter to de-clutter the page.

**Review by Eve-Parity — APPROVED** [review]

File: v3/how-to-guides/01-basics/a-pallet-integration/index.mdx (hunk 2 above)
Comment (NukeManDan) [reply v3/how-to-guides/01-basics/a-pallet-integration/index.mdx:8]: So should this be removed from **all** HTGs?

File: v3/how-to-guides/01-basics/c-constants/index.mdx (hunk 3 above)
Comment (NukeManDan) [reply v3/how-to-guides/01-basics/c-constants/index.mdx:11]: this was an artifact from the old `<Message>` mdx item

File: v3/how-to-guides/01-basics/a-pallet-integration/index.mdx (hunk 2 above)
Comment (NukeManDan) [reply v3/how-to-guides/01-basics/a-pallet-integration/index.mdx]: I went ahead and did this.

**Pull request merged**